test/test-utils/gameManager.ts

```diff
@@ -265,7 +265,7 @@ export class GameManager {
       "EncounterPhase",
       UiMode.MESSAGE,
       () => {
-        const handler = this.scene.ui.getHandler() as BattleMessageUiHandler;
+        const handler = this.scene.ui.getCurrentHandler() as BattleMessageUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        const handler = this.scene.ui.getCurrentHandler<BattleMessageUiHandler>();
```